niworkflows/anat/mni.py

```diff
@@ -173,6 +173,42 @@ def _config_ants(self, ants_settings):
                 self.norm.inputs.fixed_image_mask = op.join(
                     mni_template, '%dmm_brainmask.nii.gz' % resolution)
 
+    def _get_resolution(self):
+        resolution = self.inputs.template_resolution
+        if self.inputs.testing:
+            resolution = 2
+        return resolution
+
+    def _get_template(self):
+        get_template = getattr(getters, 'get_{}'.format(self.inputs.template))
+        mni_template = get_template()
+        return mni_template
```

Comment:
Is there a reason we create the function string, when that function just calls `get_dataset(template)`?

```Python
def _get_template(self):
    return getters.get_dataset(self.inputs.template)
```